lib/memjs/memjs.js

```diff
@@ -7,14 +7,22 @@ var hashCode = require('./utils').hashCode;
 var merge = require('./utils').merge;
 var makeExpiration = require('./utils').makeExpiration;
 var makeAmountInitialAndExpiration = require('./utils').makeAmountInitialAndExpiration;
+var nonSerialization = require('./serializers/non-serialization');
```

Comment:
I would call this (as well as the module it imports) `noSerialization` or `noopSerialization`